types/Container.dhall

```diff
@@ -1,7 +1,5 @@
 -- This expression is only suitable for the most simple container representations:
 --
--- > When you only specify a container image, you can omit the image keyword.
---
 -- TODO: Add the remaining fields as per
 -- https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer
-Text
+{ image : Text, options : Text }
```

Comment:
would it be better to make it
```suggestion
{ image : Text, options : Optional Text }
```

otherwise this will be required.

Reply:
Thanks, that's a spot on! Actually, I've though of this right during refactoring our pipeline, but forgot about it later, this week was really CI-consuming 😄